WBooks/Tabs/Library/LibraryController.swift

```diff
@@ -66,24 +66,24 @@ extension MainMenuController: UITableViewDelegate {
 }
 
 // MARK: - UITableViewDataSource
-extension MainMenuController: UITableViewDataSource {
+extension LibraryController: UITableViewDataSource {
     private func configureBooksTable() {
```

Comment:
This methods should be in the "private section" not in the extension protocol tableview data source :P